src/ORM/Marshaller.php

```diff
@@ -479,6 +481,16 @@ public function merge(EntityInterface $entity, array $data, array $options = [])
         return $entity;
     }
 
+    public function mergeTranslations($original, $value) {
```

Comment:
I'm not super excited about it, but I can't think of a better solution right now.